include/grpcpp/server_impl.h

```diff
@@ -333,9 +333,15 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
 
   std::unique_ptr<grpc_impl::ServerInitializer> server_initializer_;
 
+  // This is needed for an internal change.
+  // TODO(https://github.com/grpc/grpc/issues/19000): We should be able to move
+  // this above the private section entirely instead of inserting it here, but
+  // there's a bug.
+ protected:
```

Comment:
No protected data members according to the style guide. Can these just be methods instead?

Reply:
(and yes, I realize that we are violating that part of the style guide in a few places but we shouldn't increase the number of such places)

Reply:
Done.